force-app/main/default/classes/SOQL.cls

```diff
@@ -49,9 +49,14 @@ public with sharing class SOQL {
 	 * @return A list of Contact Ids.
 	 * Update the corresponding test class and testGetContactIdsByLastName method to cover this method
 	 */
-	//[public | private ] [static] data_type method_name (input parameters) {
-	//  The body of the method
-	//}
+	public static List<Id> getSmithJonesContacts() {
+		List<Contact> contacts = [SELECT Id FROM Contact WHERE LastName IN ('Smith', 'Jones')];
+		List<Id> contactIds = new List<Id>();
+		for (Contact c : contacts) {
```

Comment:
Try to see if you can avoid single letter variables, see if you can make the descriptive.